site/topics/numbers/number-bases.rst

```diff
@@ -418,6 +418,301 @@ Hexadecimal (Base 16)
 Converting Numbers Between Bases
 ================================
 
+* Although only three bases were discussed so far, but there is nothing stopping one from using base 5, or 123, or ...
+
+* A subscript is added to the number to specify the base when there is a potential for ambiguity
+* For example, consider the number :math:`10`
+
+    * What number is that?
+    * Is it a base ten number?
+    * Base two?
+    * Hexadecimal?
+
+
+* Ambiguity is eliminated by including the subscript
+
+    * :math:`10_{10}` means ten
+    * :math:`10_{2}` means two
+    * :math:`10_{16}` means sixteen
+
+
+* In a computing context, binary and hexadecimal are often written as ``0b10`` and ``0x10`` respectively
+
+* Notice that, with any base greater than 1, the value of :math:`10` will always be the base
+
+    * Given a base :math:`b`
+    * The second digit is always associated with the value :math:`b^{1}`
+    * And if the value in the second digit is :math:`1`, then the number is :math:`1 \times b^{1} == 1 \times b == b`
```

Comment:
This may be my already knowing what number bases are, but this feels unnecessary to point out.

Reply:
I came up with a response to justify keeping it, but upon further though, given the whole "this topic feels sloppy", there's even more of an argument to remove this.

Reply:
To be clear, I removed it.